osu.Game.Rulesets.Osu/Mods/OsuModSpunOut.cs

```diff
@@ -18,5 +23,33 @@ public class OsuModSpunOut : Mod
         public override double ScoreMultiplier => 0.9;
         public override bool Ranked => true;
         public override Type[] IncompatibleMods => new[] { typeof(ModAutoplay), typeof(OsuModAutopilot) };
+
+        private double lastFrameTime;
+        private float frameDelay;
+
+        public void ApplyToDrawableHitObjects(IEnumerable<DrawableHitObject> drawables)
+        {
+            foreach (var hitObject in drawables)
+            {
+                if (hitObject is DrawableSpinner spinner)
+                {
+                    spinner.Disc.Trackable = false;
+                    spinner.Disc.OnUpdate += d =>
+                    {
+                        if (d is SpinnerDisc s)
```

Comment:
Looks like a redundant check, but...

Reply:
Why is this required? Is this just a weird null check? I don't see how this could be false otherwise.